File: src/main/java/io/earlisreal/ejournal/dto/Portfolio.java
package io.earlisreal.ejournal.dto;

public class Portfolio {

    private Integer id;
    private String name;

    public Portfolio(int id, String name) {
        this.id = id;
        this.name = name;
    }

    public Portfolio(String name) {
        this.name = name;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: shared/src/commonMain/kotlin/io/earlisreal/ejournal/domain/model/Portfolio.kt
package io.earlisreal.ejournal.domain.model

import io.earlisreal.ejournal.domain.alpaca.AlpacaEnvironment

data class Portfolio(
    val id: Long,
    val name: String,
    val market: Market,
    val broker: Broker?,
    val credentialRef: String,
    val alpacaEnvironment: AlpacaEnvironment? = null,
)
